### PR TITLE
Providing possibility to disable telemetry for api-server for users.

### PR DESCRIPTION
**Pull request by fog1985**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by exu — APPROVED** [review]
LGTM

File: charts/api-server/templates/deployment.yaml [hunk 1]
```diff
@@ -36,6 +36,8 @@ spec:
             value: "{{ .Values.mongoDSN }}"
           - name: POSTMANEXECUTOR_URI
             value: "{{ .Values.postmanExecutorURI }}"
+          - name: TESTKUBE_TELEMETRY_DISABLED
```
Comment (nicufk) [inline charts/api-server/templates/deployment.yaml:39]: Can we have naming with enabled instead of disabled, disabled = false can create some confusion.

**Review by jasmingacic — APPROVED** [review]

File: charts/api-server/templates/deployment.yaml (hunk 1 above)
Comment (fog1985) [reply charts/api-server/templates/deployment.yaml:39]: For me it doesn't matter honestly speaking. If TESTKUBE_TELEMETRY_DISABLED = false it means that it's enabled etc. Whatever you prefer. 
@jasmingacic , if we change this name you will have to change it in api-server code-base.

**Review by nicufk — APPROVED** [review]

File: charts/api-server/templates/deployment.yaml (hunk 1 above)
Comment (jasmingacic) [reply charts/api-server/templates/deployment.yaml:39]: yes i would have to change it in the code which hasn't been merged yet https://github.com/kubeshop/testkube/pull/475

**Pull request merged**